replay-protected-sighash.md

```diff
@@ -1,12 +1,13 @@
-# BUIP-HF Digest for replay protected signature verification accross hard forks
+# BUIP-HF Digest for replay protected signature verification across hard forks
 
 Version 1.0, 2017-06-14
+Version 1.1, 2017-06-14
```

Comment:
I think we don't need to keep a long version history as a pre-amble.
Just replace the existing text with yours, that'll do fine. Git will keep the version history for us for now :-)

Also, use the "View" function on Markdown document in your branch before you do a PR. It will show you any possible layout glitches like in this case, where the versions are squashed onto one line, presumably not what you intended to happen.